plugins/mcreator-localization/help/default/entity/data_default_value.md

```diff
@@ -0,0 +1,4 @@
+This is the default value of the entity's variable.
+- **Logic**: _true_ or _false_;

```

Comment:
We usually use * not -, and don't split lines with semicolon in those lists in other tips